plugins/mobile-plugin/src/main/java/ru/sbtqa/tag/pagefactory/mobile/actions/MobilePageActions.java

```diff
@@ -50,6 +50,13 @@ private void clickViaAdb(WebElement webElement) {
         // get center point of element to tap on it
         int x = webElement.getLocation().getX() + webElement.getSize().getWidth() / 2;
         int y = webElement.getLocation().getY() + webElement.getSize().getHeight() / 2;
+        try {
+            System.out.println("sleep");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)